docs/UserGuide.adoc

```diff
@@ -113,7 +116,7 @@ Adds a new meme to Weme. Weme will copy the given image into its data storage fo
 Format: `add p/PATH [d/DESCRIPTION] [t/TAG]...` +
 
 [TIP]
-A meme can have any number of tags (including 0)
+A meme can have any number of tags (including 0). Tags can only contain letters and numbers.
```

Comment:
Since you are at it, do you want to specify that tag can only hold 35 alphanumeric characters?